package.json

```diff
@@ -11,7 +11,7 @@
     "url": "https://github.com/thejameskyle"
   },
   "scripts": {
-    "test": "mocha"
+    "test": "jest __tests__/*.js"
```

Comment:
nit
```suggestion
    "test": "jest"
```

Reply:
Good catch, let's stick with the defaults